### PR TITLE
Fustat ready for Google Fonts

### PR DESCRIPTION
**Pull request by yanone**
I'm commissioned to onboard Fustat to Google Fonts.

The fonts were in excellent shape except for the outline direction which was the wrong way around in many cases. I corrected that and also added a description file which I'd like to ask you to expand with a few short paragraphs.

Thank you.

### CODE REVIEW

File: fonts/variable/Fustat[wght].ttf
Comment (khaledhosny) [inline fonts/variable/Fustat[wght].ttf:1]: The font was not included intentionally. Built binaries should be obtained from the release artifacts.

File: DESCRIPTION.en_us.html
Comment (khaledhosny) [inline DESCRIPTION.en_us.html:1]: There is already (also rudimentary) one under `documentation/`, if it needs to be in the top directory, please move it.

File: fonts/variable/Fustat[wght].ttf
Comment (yanone) [reply fonts/variable/Fustat[wght].ttf:1]: Okay, I'll fix both of these early next week. Thank you.

**Comment by khaledhosny** [comment]
I merged the contour direction corrections manually (and fontbakery reported that `multiply` also had the wrong direction, so I fixed it), and I also moved the existing DESCRIPTION.en_us.html to top directory.

I’m waiting for Gaber to update the description and then will cut a new release. Please let u know if anything else needs to be fixed before the release. Thanks!

**Comment by yanone** [comment]
At this point I can't think of anything besides the description. It's possible that the second layer of reviewers find more issues once I make a PR to goole/fonts, but it's too early to tell.
Thank you.

**Comment by khaledhosny** [comment]
I cut a new release https://github.com/Kief-Type-Foundry/Fustat/releases/tag/v1.007, and it includes updated DESCRIPTION.en_us.html file.

**Pull request closed without merging**